influxql/ast.go

```diff
@@ -5300,9 +5309,16 @@ type Valuer interface {
 	Value(key string) (interface{}, bool)
 }
 
+// ZoneValuer is the interface that specifies the current time zone.
+type ZoneValuer interface {
```

Comment:
It's because this is an extension of the `Valuer` interface and not all `Valuer`s can have a Zone method.